activation/post_verifier.go

```diff
@@ -172,6 +194,10 @@ func (v *OffloadingPostVerifier) Verify(
 	m *shared.ProofMetadata,
 	opts ...verifying.OptionFunc,
 ) error {
+	if _, ok := v.prioritizedIds[types.BytesToNodeID(m.NodeId)]; ok {
+		v.log.Debug("prioritizing post verification", zap.Stringer("proof_node_id", types.BytesToNodeID(m.NodeId)))
+		return v.verifier.Verify(ctx, p, m, opts...)
+	}
```

Comment:
NIT: this ignores config settings in the sense that if I only use 1 worker to verify, in the worst case 1+len(v.prioritizedIds) "workers" (the others being the go routines that call `Verify`) will be used when prioritised PoSTs are verified.

We could add a second queue besides `v.jobs` that is read from first by the workers and only if it doesn't contain a pending job `v.jobs` is used to fetch the next job. This way a prioritised PoST would "jump the queue" and be the next in line over all other PoSTs.

Reply:
Thanks, I refactored the code to use 2 job channels.